expertsystem/reaction/solving.py

```diff
@@ -746,16 +562,15 @@ def get_rules_by_priority(
             # and strip away the priorities again
             return [x[0] for x in sorted_list]
 
+        arg_handler = RuleArgumentHandler()
+
         for node_id in self.__graph.nodes:
-            # currently we only have rules related to graph nodes
```

Comment:
Thinking about this comment. Did it become outdated because of the new distinction between `EdgeInteractionRule` and `InteractionRule`?

Reply:
Partly, especially the new upcoming edge rules will make it different

Reply:
Upcoming edge rules?